### DISCUSSION TITLE
CSS is not being applied to pages

### DISCUSSION THREAD

**Discussion by Hacharoui** [community/community#23567] [Other Feature Feedback, Questions, & Ideas]
<p>Hello there, i have recently crafted a webpage using HTML and CSS and for some reason my CSS is not being applied to some of my HTML pages in the recipes folder when i open up the webpage through GitHub. I am quite new to GitHub so i would highly appreciate if someone can shed some light on this.</p>
<p>GitHub files.</p><aside class="onebox allowlistedgeneric" data-onebox-src="https://github.com/driftblade1/Odin-Recipies-Improved-CSS">
  <header class="source">
      <img src="https://github.githubassets.com/favicons/favicon.svg" class="site-icon" width="32" height="32">

      <a href="https://github.com/driftblade1/Odin-Recipies-Improved-CSS" target="_blank" rel="noopener nofollow ugc">GitHub</a>
  </header>

  <article class="onebox-body">
    <div class="aspect-image" style="--aspect-ratio:690/345;"><img src="https://user-images.githubusercontent.com/79722131/181077948-7e1157bf-95e1-497e-9d3f-089a1116e2c3.png" class="thumbnail" width="690" height="345" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/e/7/e794e5e255c2278b41f5049d650a02244e1b03cf_2_690x345.png, https://aws1.discourse-cdn.com/github/optimized/3X/e/7/e794e5e255c2278b41f5049d650a02244e1b03cf_2_1035x517.png 1.5x, https://aws1.discourse-cdn.com/github/original/3X/e/7/e794e5e255c2278b41f5049d650a02244e1b03cf.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/e/7/e794e5e255c2278b41f5049d650a02244e1b03cf_2_10x10.png"></div>

<h3><a href="https://github.com/driftblade1/Odin-Recipies-Improved-CSS" target="_blank" rel="noopener nofollow ugc">GitHub - driftblade1/Odin-Recipies-Improved-CSS: Upgrading my previous...</a></h3>

  <p>Upgrading my previous webpage with CSS and using CSS fundementals.  - GitHub - driftblade1/Odin-Recipies-Improved-CSS: Upgrading my previous webpage with CSS and using CSS fundementals.</p>

  </article>

  <div class="onebox-metadata">
    
    
  </div>

  <div style="clear: both"></div>
</aside>

<p>Webpage:<br>
<a href="https://driftblade1.github.io/Odin-Recipies-Improved-CSS/" class="onebox" target="_blank" rel="noopener nofollow ugc">https://driftblade1.github.io/Odin-Recipies-Improved-CSS/</a></p>

**Comment by jfullstackdev** [comment, marked as answer]
<p>because it’s inside another folder <code>recipes</code> and relative to that, there is no <code>CSS</code> folder there,</p>
<p><img src="https://user-images.githubusercontent.com/47092464/181077951-58986fb0-de9e-418a-9270-ac19cdca629b.png" alt="image" data-base62-sha1="kGnsu0gPg0blg50ZfwYP6bBtRJH" width="352" height="44"></p>
<p>if you are confused, try absolute URL,</p>
<p>for example if you want to link at your recipe the <code>style2.css</code>, change the link to</p>
<p><code>https://driftblade1.github.io/Odin-Recipies-Improved-CSS/CSS/style2.css</code></p>

**Comment by Hacharoui** [comment]
<p>Thank buddy, i think the absolute URL method clears up confusion !</p>
